.github/workflows/ci.yml

```
@@ -0,0 +1,51 @@
+# This workflow will install dependencies, run tests and the black-formatter
+
+name: om3_scripts
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  formatting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Code formatting
+      uses: psf/black@stable
+      with:
+        options: "--check --verbose --diff"
+
+  build:
```

```suggestion
  test:
```
Take or leave, but this might be more appropriate for what is being done in this task